hr_course/views/hr_employee_views.xml

```diff
@@ -16,6 +16,19 @@
                     <field name='courses_ids' invisible="1" />
                 </button>
             </div>
+            <xpath expr="//notebook" position="inside">
```

Comment:
Why the notebook? It was already on a button box, as an employee could have a lot of courses. I think it is better to keep it as it is.

Reply:
@etobella  sorry, i missed that there is a button to see the courses.
@gurneyalex  this is the requirement coming from our customer. (card 103) Should be moved to internal module maybe?